lib/hanami/action/flash.rb

```diff
@@ -92,10 +114,10 @@ def clear
         #
```

Comment:
Do you have any idea about the `FIXME` on line 112? Might be a good time to tackle that if you can figure it out. It might not be relevant anymore with the refactoring that you did?

Reply:
Unfortunately, I have no idea about it. Maybe @jodosha can give some information